pom.xml

```diff
@@ -72,22 +72,22 @@
         <scala.binary.version>2.12</scala.binary.version>
 
         <avro.version>1.11.3</avro.version>
-        <guava.version>32.1.3-jre</guava.version>
+        <guava.version>33.0.0-jre</guava.version>
 
         <!-- versions from geomesa -->
         <gt.version>28.2</gt.version>
         <accumulo.version>2.0.1</accumulo.version>
         <thrift.version>0.12.0</thrift.version>
         <zookeeper.version>3.7.2</zookeeper.version>
         <hadoop.version>3.3.6</hadoop.version>
-        <hbase.version>2.5.6-hadoop3</hbase.version>
+        <hbase.version>2.5.7-hadoop3</hbase.version>
         <hbase-1.version>1.4.14</hbase-1.version>
         <kafka.version>2.8.2</kafka.version>
         <zkclient.version>0.11</zkclient.version>
         <postgres.version>42.7.0</postgres.version>
 
         <curator.version>4.3.0</curator.version>
-        <netty.version>4.1.101.Final</netty.version>
+        <netty.version>4.1.101.Final</netty.version>  <!-- NOTE: versions 4.1.102.Final+ break ConvertToGeoFileTest -->
```

Comment:
we should probably figure out why and fix this...

Reply:
Yeah, I looked into it a bit - it seems to be an issue between Apache Arrow and the most recent version(s) of Netty that others have encountered. Here's [an example](https://github.com/apache/arrow/issues/39265).

The `java.lang.NoSuchFieldError: chunkSize` error in the 2nd screenshot is the same one I was getting.

Reply:
ah, so we probably will need to upgrade arrow versions... d'oh

Reply:
That was my guess as well, but I wasn't sure what that might break downstream and was honestly too scared to find out! :grimacing: